_saas-integrations/pinterest-ads/v1/pinterest-ads-v1.md

```diff
@@ -0,0 +1,150 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Pinterest Ads (vVERSION)
+permalink: /integrations/saas/pinterest-ads ## Add if there are multiple versions: /vVERSION
+keywords: pinterest-ads, integration, schema, etl pinterest-ads, pinterest-ads etl, pinterest-ads schema
+layout: singer
+# input: false
+
+key: "pinterest-ads-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "pinterest-ads"
+display_name: "Pinterest Ads"
+
+singer: true
+status-url: ""
+
+tap-name: "Pinterest Ads" ## Ex: Intercom, not intercom
+repo-url: https://github.com/singer-io/tap-pinterest-ads
+
+this-version: "1"
+
+api: |
+  [Pinterest Ads API V3](https://developers.pinterest.com/docs/redoc/adsreporting/){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Standard"
+
+api-type: "platform.pinterest-ads"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: |
+      A {{ integration.display_name }} OAuth App. If you don't have one, refer to {{ integration.display_name }}'s docs [here](https://developers.pinterest.com/docs/redoc/adsreporting/#section/User-Authorization/Start-the-OAuth-flow-(explicit-server-side)) for instructions on how to create one.
```

Comment:
The parentheses in the original URL were breaking the Markdown, so I changed it to the top-level section on the linked page. 

```suggestion
      A {{ integration.display_name }} OAuth App. If you don't have one, refer to {{ integration.display_name }}'s [documentation](https://developers.pinterest.com/docs/redoc/adsreporting/#section/User-Authorization){:target="new"} for instructions on how to create one.
```